common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -28,7 +28,7 @@ class @Problem
     problem_prefix = @element_id.replace(/problem_/,'')
     @inputs = @$("[id^='input_#{problem_prefix}_']")
     @$('div.action button').click @refreshAnswers
-    @questionTitle = @$('.problem-header')
+    @questionTitle = @$("[id^='#{@id}-problem-title']")
```

Comment:
Was it necessary to use an ID vs a class? I was assuming a class would work because the selector would be scoped within the individual problem-- is this not how it is behaving?